packages/logger/src/logger-event.ts

```diff
@@ -5,7 +5,14 @@ export declare type LoggerEvent =
     | 'HealthCheck'
     | 'BatchScanRequestSubmitted'
     | 'ScanRequestSubmitted'
+    | 'ScanTasksQueued'
     | 'ScanTaskStarted'
     | 'ScanTaskCompleted'
     | 'ScanTaskSucceeded'
     | 'ScanTaskFailed';
+
+[
```

Comment:
no sure why we need this

Reply:
sorry, some random text got in somehow...

Reply:
removed.